### PR TITLE
new speed score formula

### PR DESCRIPTION
**Pull request by BongleXD**

### CODE REVIEW

File: src/main/kotlin/skytils/skytilsmod/features/impl/dungeons/ScoreCalculation.kt [hunk 1]
```diff
@@ -66,7 +64,7 @@ object ScoreCalculation {
         "F4" to FloorRequirement(.6, 12 * 60),
         "F5" to FloorRequirement(.7),
         "F6" to FloorRequirement(.85, 12 * 60),
-        "F7" to FloorRequirement(speed = 14 * 60),
```
Comment (My-Name-Is-Jeff) [inline src/main/kotlin/skytils/skytilsmod/features/impl/dungeons/ScoreCalculation.kt:69]: Supposedly they changed the requirement to 14 minutes

**Review by My-Name-Is-Jeff — CHANGES_REQUESTED** [review]

File: src/main/kotlin/skytils/skytilsmod/features/impl/dungeons/ScoreCalculation.kt (hunk 1 above)
Comment (BongleXD) [reply src/main/kotlin/skytils/skytilsmod/features/impl/dungeons/ScoreCalculation.kt:69]: I played couple of f7 runs but still 12 mins

Comment (My-Name-Is-Jeff) [reply src/main/kotlin/skytils/skytilsmod/features/impl/dungeons/ScoreCalculation.kt:69]: bruh what

Comment (My-Name-Is-Jeff) [reply src/main/kotlin/skytils/skytilsmod/features/impl/dungeons/ScoreCalculation.kt:69]: I think they said on 1/6 they rolled it out

https://cdn.discordapp.com/attachments/881403326938353684/929449078574706728/unknown.png

Comment (My-Name-Is-Jeff) [reply src/main/kotlin/skytils/skytilsmod/features/impl/dungeons/ScoreCalculation.kt:69]: just checked, it's 12

**Pull request merged**